source/patchcanvas_theme.py

```diff
@@ -337,14 +338,14 @@ def __init__(self, idx):
             self.port_font_state = QFont.Normal
             self.port_mode = self.THEME_PORT_SQUARE
 
-            self.port_audio_jack_pen = QPen(QColor(35, 61, 99), 0)
```

Comment:
why are you removing the custom colors?

Reply:
Please look above: this line is just moved out of theme-specific block. Color doesn't make sence, since it is placeholder for invisible line. Unfortunally, simple QPen(Qt.NoPen) has linewidth 1px, and alternative constructors require color, brush or like in begining, so i used "dumb" color.